packages/model/addon/-private/model.js

```diff
@@ -423,13 +394,32 @@ const Model = EmberObject.extend(DeprecatedEvented, {
     @type {Boolean}
     @readOnly
   */
-  isError: isError,
+  get isError() {
+    if (REQUEST_SERVICE) {
+      let errorReq = this._errorRequests[this._errorRequests.length - 1];
+      if (!errorReq) {
+        return false;
+      } else {
+        return true;
+      }
+    }
+    let meta = getRecordMeta(this);
+    return get(meta, 'isError') || false;
+  }
+  set isError(v) {
+    if (REQUEST_SERVICE && DEBUG) {
+      throw new Error(`isError is not directly settable when REQUEST_SERVICE is enabled`);
+    } else {
+      let meta = getRecordMeta(this);
+      set(meta, 'isError', v);
+    }
+  }
```

Comment:
My best guess is b/c Ember's internals relies on observers for query param bindings, we are getting the "used previously in the same computation" error after it was set.

https://github.com/emberobserver/client/blob/2c3cc0f1a1547e716477c9e18a40ebe48a6490a4/app/controllers/admin/size-calculation-results/index.js#L8

Reply:
could be, we may need to return the value from the setter or some such (computeds have funny semantics for set)

Reply:
partner test is no longer failing